plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -1081,6 +1124,11 @@ app.route('/manage/db/aggregate/:dbs/:collection', 'dbs', function(db, collectio
     }
 });
 
+app.route('/manage/db/task/*task', 'task_id', function(task_id) {
+    this.dbviewerView._task = task_id;
```

Comment:
I think you need to set ._task to null on all other db viewer paths, or else if you go to dbviewer view, it would still have the task defined, even if you are not going for the task